libkbfs/disk_block_cache.go

```diff
@@ -466,31 +463,41 @@ func (cache *DiskBlockCacheStandard) encodeBlockCacheEntry(buf []byte,
 	return cache.config.Codec().Encode(&entry)
 }
 
-// Get implements the DiskBlockCache interface for DiskBlockCacheStandard.
-func (cache *DiskBlockCacheStandard) Get(ctx context.Context, tlfID tlf.ID,
-	blockID kbfsblock.ID) (buf []byte,
-	serverHalf kbfscrypto.BlockCryptKeyServerHalf, hasPrefetched bool,
-	err error) {
+// checkAndLockCache checks whether the cache is started, then locks it based
+// on the provided flag, leaving the cache locked.
+func (cache *DiskBlockCacheStandard) checkCacheLocked(method string) error {
 	select {
 	case <-cache.startedCh:
 	default:
 		// If the cache hasn't started yet, pretend like no blocks exist.
-		return nil, kbfscrypto.BlockCryptKeyServerHalf{}, false,
-			DiskCacheStartingError{"Get"}
+		return DiskCacheStartingError{method}
 	}
-	cache.lock.RLock()
-	defer cache.lock.RUnlock()
 	// shutdownCh has to be checked under lock, otherwise we can race.
 	select {
 	case <-cache.shutdownCh:
-		return nil, kbfscrypto.BlockCryptKeyServerHalf{}, false,
-			DiskCacheClosedError{"Get"}
+		return DiskCacheClosedError{method}
 	default:
 	}
 	if cache.blockDb == nil {
+		return DiskCacheClosedError{"Get"}
+	}
+	return nil
+}
+
+// Get implements the DiskBlockCache interface for DiskBlockCacheStandard.
+func (cache *DiskBlockCacheStandard) Get(ctx context.Context, tlfID tlf.ID,
+	blockID kbfsblock.ID) (buf []byte,
+	serverHalf kbfscrypto.BlockCryptKeyServerHalf, hasPrefetched bool,
+	err error) {
+
```

Comment:
None of the rest of the code does it though. For me, inconsistency is way worse for readability.  I find it confusing because I think there's supposed to be real code above the blank line, and I get distracted looking for it.